cfme/tests/intelligence/reports/test_views.py

```diff
@@ -1,41 +1,38 @@
 # -*- coding: utf-8 -*-
 import pytest
+from widgetastic_patternfly import CandidateNotFound
 
 from cfme import test_requirements
-from cfme.cloud.provider.ec2 import EC2Provider
-from cfme.cloud.provider.openstack import OpenStackProvider
-from cfme.infrastructure.provider.rhevm import RHEVMProvider
-from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.common.provider import BaseProvider
+from cfme.markers.env_markers.provider import ONE
 from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.blockers import BZ
 
 pytestmark = [
     pytest.mark.tier(3),
     test_requirements.report,
-    pytest.mark.usefixtures('setup_provider'),
-    pytest.mark.provider([OpenStackProvider, EC2Provider, RHEVMProvider, VMwareProvider],
-                         scope='module')
+    pytest.mark.usefixtures("setup_provider"),
+    pytest.mark.provider([BaseProvider], selector=ONE, scope="module"),
 ]
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope="module")
 def report(appliance):
-    # TODO parameterize on path, for now test infrastructure reports
     report = appliance.collections.reports.instantiate(
         type="Configuration Management",
         subtype="Hosts",
-        menu_name="Virtual Infrastructure Platforms"
+        menu_name="Virtual Infrastructure Platforms",
     ).queue(wait_for_finish=True)
     yield report
     if report.exists:
         report.delete()
 
 
 @pytest.mark.rhv3
-@pytest.mark.parametrize('view_mode', ['Hybrid View', 'Graph View', 'Tabular View'])
-@pytest.mark.meta(blockers=[BZ(1401560)])
+@pytest.mark.parametrize("view_mode", ["Hybrid View", "Graph View", "Tabular View"])
```

Comment:
Optional:
@valaparthvi I think better to set proper ids for this parametrization in test name ["Hybrid view"] will come. Totally optional but you can separate  by `-` or `_`.